docusaurus.config.js

```diff
@@ -106,7 +114,8 @@ module.exports = {
       {
         docs: {
           sidebarPath: require.resolve('./sidebars.js'),
-          routeBasePath: "/",
+          path: 'docs/general',
+          routeBasePath: '/general', // the links to /tools break when this is '/'
```

Comment:
we could omit it, and it would default to /docs
i guess for now that would be feasible?